ads/mediaimpact.js

```diff
@@ -39,7 +48,7 @@ export function mediaimpact(global, data) {
     view: 'm',
     async: true,
   };
-  loadScript(global, 'https://ec-ns.sascdn.com/diff/251/divscripte/amp.js?dom=' + window.context.location.host, () => {
+  loadScript(global, 'https://ec-ns.sascdn.com/diff/251/pages/amp_default.js', () => {
```

Comment:
THX, I thought we added the root here instead. Then I have to change this in _config.js either. I want to reduced requests to speed up the integration a little bit...